src/core/lib/surface/call.cc

```diff
@@ -814,20 +813,15 @@ grpc_error_handle FilterStackCall::Create(grpc_call_create_args* args,
     *composite = grpc_error_add_child(*composite, new_err);
   };
 
-  Arena* arena;
   FilterStackCall* call;
   grpc_error_handle error;
   grpc_channel_stack* channel_stack = channel->channel_stack();
-  size_t initial_size = channel->CallSizeEstimate();
-  global_stats().IncrementCallInitialSize(initial_size);
   size_t call_alloc_size =
       GPR_ROUND_UP_TO_ALIGNMENT_SIZE(sizeof(FilterStackCall)) +
       channel_stack->call_stack_size;
 
-  std::pair<Arena*, void*> arena_with_call = Arena::CreateWithAlloc(
```

Comment:
Didn't we originally add `CreateWithAlloc()` for performance reasons, since we wanted to create the initial allocation at the same time that we created the alloc?  Do we no longer need that?

Conversely, if we don't need that anymore, should we remove this method from the `Arena` API?

Reply:
We might want it in the future, but for now I'm happy to take a (minor enough we won't measure it) regression in order to get the refactoring that needs doing underway :)